packages/jsii-kernel/lib/kernel.ts

```diff
@@ -596,6 +619,8 @@ export class Kernel {
                     });
                 }
             });
+        } else if (!methodInfo && Object.keys(obj).includes(methodName)) {
+            throw new Error('boom');
```

Comment:
Congratulations - you earned yourself a distracting gif of confusing!

![](https://media.giphy.com/media/CiYImHHBivpAs/giphy.gif)

Reply:
Oops. Debug code.